_drgn.pyi

```diff
@@ -1738,6 +1738,13 @@ class StackFrame:
         dictionary with the register names as keys.
         """
         ...
+    def locals(self) -> List[str]:
+        """
+        Get a list of all local variable names in local scopes for this stack
+        frame. Not all names may have present values, but they can be used with
+        __getitem__() to check.
```

Comment:
Those less familiar with the Python data model would probably recognize `[]` rather than `__getitem__`. We do it like this elsewhere in the documentation:

```suggestion
        the :meth:`[] <.__getitem__>` operator to check.
```